### DISCUSSION TITLE
GPU Core/Memory Clock Offset maximum value?

### DISCUSSION THREAD

**Discussion by barry7wu** [seerge/g-helper#1115] [General]
Many thanks to seerge for bringing us the wonderful G-Helper program!

My laptop is GA402XV using GHelper v0.112. When using the new Undervolt GPU feature, I noticed that the maximum value for GPU Core/Memory Clock Offset is +250MHz. Is this value too conservative? (In my memory, Memory Clock Offset usually goes beyond +1000MHz)

![image](https://github.com/seerge/g-helper/assets/137846987/3a9c8c1d-6cf9-4ab2-bd14-e2f508dace77)

**Comment by seerge** [comment]
@barry7wu hello,

You can unlock higher values if you want via config

https://github.com/seerge/g-helper#higher-maximum-gpu-clock--memory-offsets

**Reply by barry7wu** [reply]
Oh! I didn't notice the comment. Thank you for your response.
